src/index.js

```diff
@@ -79,4 +79,4 @@ tasks.forEach((task) => {
 });
 saveData();
 document.querySelector('.add-btn').addEventListener('click', addToList);
-document.querySelector('.removeButton').addEventListener('click', removeList);
\ No newline at end of file
+document.querySelector('.removeButton').addEventListener('click', removeList);
```

Comment:
- [ ]  You need to have new javascript file for the new functionality added 
![image](https://github.com/LethaboMot/weppack2/assets/62705947/0a12802d-b47f-4f23-b396-d47fe857f013)